cli_reference/developer_cli_odo/deleting-applications.adoc

```diff
@@ -0,0 +1,8 @@
+[id='deleting-applications']
+= Deleting applications
+include::modules/developer-cli-odo-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: deleting-applications
+toc::[]
+
```

Comment:
You need an intro statement for the assembly.

Reply:
Added.